src/App.js

```diff
@@ -13,6 +14,7 @@ function App() {
         <Routes>
           <Route path="/" element={<ReviewList />} />
           <Route path="/reviews/:category" element={<ReviewList />} />
+          <Route path="/review/:review_id" element={<SingleReview />} />
```

Comment:
In terms of constructing your paths, you have to be reeeeally careful. Remember our lessons on RESTful APIs and how we construct paths. It might make more sense to use `/reviews/:review_id` than the singular "review".